### PR TITLE
largest-series-product: Add parameter to exercise placeholder (#567)

### LINKED ISSUE #567: largest-series-product: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by iandexter**
Signed-off-by: Ian Dexter D Marquez <iandexter@users.noreply.github.com>

Closes #567

### CODE REVIEW

File: exercises/largest-series-product/largest_series_product.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-def largest_product():
+def largest_product(digits):
```
Comment (m-a-ge) [inline exercises/largest-series-product/largest_series_product.py]: According to test cases https://github.com/exercism/python/blob/master/exercises/largest-series-product/largest_series_product_test.py `largest_product` takes two parameters

Please correct your suggestion

Comment (iandexter) [reply exercises/largest-series-product/largest_series_product.py]: Pushed 6714093 with the change. Thanks!

**Comment by m-a-ge** [comment]
Merged, thanks, @iandexter

**Comment by m-a-ge** [comment]
One more thing, it's a good practice to squash your commits https://github.com/exercism/docs/blob/master/contributing/git-basics.md#squashing. It isn't a big deal since I can do it myself, but it would be awesome to just click on 'Merge pull request' button 😃

**Pull request merged**